Wangscape/noise/module/CornerCombinerBase.h

```diff
@@ -4,6 +4,8 @@ namespace noise
 {
 namespace module
 {
+// Default power for the noise::module::CornerCombinerBase module.
```

Comment:
For classes derived from `noise::module::Module`, I'd prefer to imitate the libnoise coding style (hence the `PascalCase` method names). That includes comments on constant values whose purpose should be obvious. ([example](https://github.com/qknight/libnoise/blob/master/src/noise/module/rotatepoint.h#L45))